include/SFML/Graphics/Texture.hpp

```diff
@@ -272,13 +271,11 @@ class SFML_GRAPHICS_API Texture : GlResource
     /// texture was not previously created.
     ///
     /// \param pixels Array of pixels to copy to the texture
-    /// \param width  Width of the pixel region contained in \a pixels
-    /// \param height Height of the pixel region contained in \a pixels
-    /// \param x      X offset in the texture where to copy the source pixels
-    /// \param y      Y offset in the texture where to copy the source pixels
+    /// \param size   Width and height of the pixel region contained in \a pixels
+    /// \param offset Offset in the texture where to copy the source pixels
```

Comment:
> Offset in the texture where to copy the source pixels

This sounds confusing (I know it was before). Is this source or destination?
Maybe we should also rename parameters for clarity.

Reply:
Reading through the source, `offset` appears to representation a destination, not a source.

Reply:
Should it be renamed to destination everywhere?

Reply:
I like that idea